.github/workflows/tests.yml

```diff
@@ -74,8 +74,8 @@ jobs:
         WORKDIR=$(readlink -f pkg/jobqueue/dbjobqueue/schemas)
         pushd $(mktemp -d)
         go mod init temp
-        go get -u github.com/jackc/tern
-        go run github.com/jackc/tern migrate -m "$WORKDIR"
+        go install github.com/jackc/tern@latest
```

Comment:
Wonder if we shouldn't pin a well known version, but not in this PR.

Reply:
How is this done in prod, @croissanne ? Would be great to match these.